ui/app/components/send/send.component.js

```diff
@@ -11,6 +11,9 @@ import {
 import SendHeader from './send-header/'
 import SendContent from './send-content/'
 import SendFooter from './send-footer/'
+import BigNumber from 'bignumber.js'
+
+window.BigNumber = BigNumber
```

Comment:
why do we need to put `BigNumber` on `window`? I'm guess this was just for local development/testing purposes?